doc/changelog.rst

```diff
@@ -1,6 +1,37 @@
 Changelog
 =========
 
+Changes in Version 4.2
+----------------------
+
+Bug fixes
+.........
+
+- Fixed a bug where :meth:`~pymongo.collection.Collection.estimated_document_count`
+  would fail with a "CommandNotSupportedOnView" error on views (`PYTHON-2885`_).
+
+Unavoidable breaking changes
+............................
+
+- :meth:`~pymongo.collection.Collection.estimated_document_count` now always uses
+  the `count`_ command. Due to an oversight in versions 5.0.0-5.0.8 of MongoDB,
+  the count command was not included in v1 of the :ref:`versioned-api-ref`.
```

Comment:
This should be "V1" to match the name in `pymongo.server_api.ServerApiVersion`

Reply:
Fixed.